File: src/Application.Operation/Features/Employee/Exams/DTOs/ExamPartDto.cs
namespace Application.Operation.Features.Employee.Exams.DTOs;

public sealed class ExamPartDto
{
    public int PartNo { get; set; }
    public string TitleAr { get; set; } = "";
    public string? TitleEn { get; set; }
    public int DurationMinutes { get; set; }
    public decimal? QualificationScore { get; set; }
    public List<ExamCategoryDto> Categories { get; set; } = [];
}

File: src/Tawtheef.Websites/operations-web/src/app/pages/user/employee/exams-management/models/exam-part.dto.ts
import { ExamCategoryDto } from "./exam-category.dto";

export interface ExamPartDto {
  partNo: number;
  titleAr: string;
  titleEn: string | null;
  durationMinutes: number;
  qualificationScore: number | null;
  categories: ExamCategoryDto[];
}